### PR TITLE
Updated Clean Architecture template to make it more similar to `scala-seed.g8`

### PR DESCRIPTION
**Pull request by ZheniaTrochun**

### CODE REVIEW

File: src/main/g8/build.sbt [hunk 1]
```diff
@@ -1,42 +1,82 @@
...
+      .value
+      .filterNot(_.contains("wartremover"))
+      .filterNot(Scalac.Lint.toSet)
+      .filterNot(Scalac.FatalWarnings.toSet) :+ "-Wconf:any:silent"
+  },
+  Test / console / scalacOptions :=
+    (Compile / console / scalacOptions).value,
+)
 
 lazy val oneToOneClasspathDependencies: String =
```
Comment (agilesteel) [inline src/main/g8/build.sbt]: If you `import Util._` it has `Cctt` which does the same thing so you can throw this one out.

File: src/main/g8/project/plugins.sbt [hunk 2]
```diff
@@ -1 +1,16 @@
...
+addDependencyTreePlugin
+
+addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.9.34")
+addSbtPlugin("com.timushev.sbt" % "sbt-rewarn" % "0.1.3")
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.6.1")
+addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.2")
+addSbtPlugin("io.spray" % "sbt-revolver" % "0.9.1")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.6")
+addSbtPlugin("org.wartremover" % "sbt-wartremover" % "2.4.16")
+addSbtPlugin("org.foundweekends.giter8" % "sbt-giter8-scaffold" % "0.13.1")
```
Comment (agilesteel) [inline src/main/g8/project/plugins.sbt]: Please add an empty line.

File: src/main/g8/.scalafix.conf [hunk 3]
```diff
@@ -0,0 +1,41 @@
+rules = [
```
Comment (agilesteel) [inline src/main/g8/.scalafix.conf:1]: This is not the latest version. It causes issues. Please download the new file.

File: src/main/g8/.scalafmt.conf [hunk 4]
```diff
@@ -0,0 +1,129 @@
+version = "3.4.0"
```
Comment (agilesteel) [inline src/main/g8/.scalafmt.conf:1]: This is not the latest version. It causes issues. Please download the new file.

**Review by agilesteel — CHANGES_REQUESTED** [review]
Thank you for this! Please make sure that the project generated by this is well formatted. You can do this by making sure that `sbt styleCheck` does not bark at you. Also make sure that there are no warnings.

This will help you with the diffs:
```bash
git init
git add .
git commit -m "Initial commit"
sbt styleFix
git diff
```

**Comment by ZheniaTrochun** [comment]
Thank you for your comments!
I've fixed formatting and made requested changes.
Also the style check passes without any problems in a newly generated project.

**Review by agilesteel — CHANGES_REQUESTED** [review]
Awesome, I just noticed one last thing: the directories under `.g8/uescase` need to be renamed (they need the numberes).

```
core -> 02-core
delivery -> 03-delivery
persistence -> 03-persistence
main -> 04-main
```
I think it's the last thing left to do. I'll merge it afterward. Thanks again for doing this!

**Comment by ZheniaTrochun** [comment]
Good catch! I've renamed those folders too.

**Comment by agilesteel** [comment]
Awesome!

**Pull request merged**